net/instaweb/rewriter/process_context.cc

```diff
@@ -58,7 +58,7 @@ ProcessContext::ProcessContext()
   // url/url_util.cc lazily initializes its "standard_schemes" table in a
   // thread-unsafe way and so it must be explicitly initialized prior to thread
   // creation, and explicitly terminated after thread quiescence.
-  url::Initialize();
+  //url::Initialize();
```

Comment:
Why was this commented out? This seems risky, per comment.

Reply:
The method no longer exists, as we moved googlurl versions. I’ll look into if there’s an alternative we should use; thanks for pointing this out as I needed that reminder.

Reply:
I think we're good here. I double checked, and I couldn't find an equivalent in the latest version, so I think it is safe to assume this is no longer needed. I cleaned this up, and added some more cleanup + a fiasco fix in https://github.com/apache/incubator-pagespeed-mod/pull/1998/commits/cca0fadb5f25fb46bf2e32ed94eee46ec85d17fe (based on querying code level todo's I left in the last change).

Reply:
Reading the comment I left above about thread-unsafe; is it worth checking whether the latest version self-initializes in a thread-safe way?

The other thing I'm wondering about is between unit tests, does it leave threads running that might make things less deterministic. But if we can get tests to pass then I think it's worth committing, and we can always figure out how to revisit if needed.

Reply:
From what I understood there's now a method to "freeze" modifying the scheme registry. If we want to be pedantic we could call that here to ensure no-one attempts to modify this registry scheme later in a potentially thread unsafe manner. (But we're not using that functionality).

Reply:
(ps: the c++ tests are passing, currently initiating a tsan-enabled run to check if that shows complaints)